04-rstudio-tour.Rmd

```diff
@@ -5,34 +5,6 @@
 ottrpal::set_knitr_image_path()
 ```
 
-In this chapter we will talk about a very useful R-related tool called RStudio. RStudio is an environment for using R that can be extremely helpful for writing code and making your analyses reproducible. 
-
-```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "A Tour of RStudio, Learning Objectives are to be able to, Recognize why RStudio is useful, Install RStudio and get started with it, Navigate RStudio"}
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Major point!! example image"}
```

Comment:
Make sure to merge main into here to get it up to date before you merge this PR.